package.json

```diff
@@ -3,10 +3,14 @@
   "version": "0.4.0",
   "private": true,
   "dependencies": {
+    "@date-io/date-fns": "1.x",
+    "@date-io/moment": "^2.0.1",
     "@material-ui/core": "^4.7.0",
     "@material-ui/icons": "^4.5.1",
     "@material-ui/lab": "^4.0.0-alpha.33",
+    "@material-ui/pickers": "^3.2.8",
     "@material-ui/styles": "^4.6.0",
+    "date-fns": "^2.9.0",
```

Comment:
Actually, one more note about deps:
date-fns: please don't use it. We already have moment and use that, and it's compatible with material-ui/pickers, so really no need to have it. Same with @date-io/date-fns.

In general, please favour using existing deps to adding new ones.